docs/index.rst

```diff
@@ -6,6 +6,13 @@
 D-Wave Binary CSP
 =================
 
+.. deprecated:: 0.3.1
+
+    ``dwavebinarycsp`` is deprecated and will be removed in Ocean 10.
+    For solving problems with constraints, we recommand using the hybrid
+    solvers in the Leap service.
```

Comment:
```suggestion
    solvers in the Leap&trade; service.
```

Import note: change this over to [sdk_index file](https://docs.ocean.dwavesys.com/en/stable/docs_binarycsp/sdk_index.html) for the SDK

Reply:
Added it to the README but not the docs after discussion with @JoelPasvolsky